skill.json

```diff
@@ -1,31 +1,41 @@
 {
-  "manifest": {
-    "publishingInformation": {
-      "locales": {
-        "en-US": {
-          "summary": "Alexa Skill For Rocket.Chat",
-          "examplePhrases": [
-            "Alexa, Open Rocketchat",
-            "Alexa, Begin Rocketchat",
-            "Alexa, Begin Rocketchat"
-          ],
-          "name": "Rocket.Chat",
-          "description": "Sample Full Description"
-        }
-      },
-      "isAvailableWorldwide": true,
-      "testingInstructions": "Sample Testing Instructions.",
-      "category": "COMMUNICATION",
-      "distributionCountries": []
-    },
-    "apis": {
-      "custom": {
-        "endpoint": {
-          "sourceDir": "lambda/custom",
-          "uri": "ask-custom-rocket-chat-default"
-        }
-      }
-    },
-    "manifestVersion": "1.0"
-  }
+    "manifest": {
+        "publishingInformation": {
+            "locales": {
+                "en-US": {
+                    "summary": "Alexa Skill For Rocket.Chat",
+                    "examplePhrases": [
+                        "Alexa, Open Rocketchat",
+                        "Alexa, Begin Rocketchat",
+                        "Alexa, Begin Rocketchat"
```

Comment:
Please change it to "Alexa, Launch Rocketchat"

Reply:
Of couse, thank you